### ISSUE TITLE
[Assistive tech and device support] Keyboard navigation order isn't meaningful or sequential. (11.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#52077]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Enrollment verification

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Keyboard navigation order isn't meaningful or sequential.

### Issue Details
When advancing through the verification workflow, keyboard focus isn't moved back to the top of the form at each new step. Instead, focus seems to stay on the Continue button. That creates a few issues:
- If a user accidentally presses [Enter] without realizing where focus is, they may trigger an action they didn't intend.
- Keyboard-only users have to backtrack multiple [Tab] stops to find the right form controls.
- For screen reader users, the next thing they hear if they press [Tab] is an element at the bottom of the page outside the form. Easy for a user to get lost trying to figure out how to get focus back to where it should be.

### Link, screenshot or steps to recreate
Steps to reproduce:
1. As test user 301, select yes for November.
2. Press Continue.
3. Press [Tab] to move to the next focusable element.
4. Note that the element that receives focus is outside the context of the form.
5. Press [Shift] + [Tab] four times to move focus back to where it should be.

### VA.gov Experience Standard
[Category Number 11, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.3_A

### Platform Recommendation
On most multi-step forms on VA.gov when advancing to the next step, focus is set to the "Step X of Y" heading above the form content. This provides immediate context for screen reader users to know where they are, and ensures that the next [Tab] stop is the first interactive element of that step of the form.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by btruongafs** [comment]
Issue resolved and deployed to staging.
